x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/create_connector_flyout/index.test.tsx

```diff
@@ -402,10 +402,10 @@ describe('CreateConnectorFlyout', () => {
         />
       );
       await act(() => Promise.resolve());
-      expect(queryByText(technicalPreviewBadgeProps.label)).not.toBeInTheDocument();
+      expect(queryByText(betaBadgeProps.label)).not.toBeInTheDocument();
     });
 
-    it('does not show tech preview badge when isExperimental is false', async () => {
+    it('does not show beta badge when isExperimental is false', async () => {
```

Comment:
I think the test descriptions should not change. I added these to ensure we don't confuse a future beta badge with tech preview badge, esp. since the variable names are not consistent.